src/components/Icon/Icon.stories.tsx

```diff
@@ -1,36 +1,40 @@
 import styled from "@emotion/styled";
-import { withKnobs, select, color } from "@storybook/addon-knobs";
 import { size } from "constants/tokens";
 import Icon, { glyphs, Size } from "./index";
 
+const Sizes = {
+  [Size.Small]: 14,
+  [Size.Default]: 16,
+  [Size.Large]: 20,
+  [Size.XLarge]: 24,
+};
+
 export default {
-  title: "Spruce Icons",
-  decorators: [withKnobs],
+  title: "Components/Icons",
   component: Icon,
+  args: {
+    color: "#000000",
+    size: Sizes[Size.Default],
+    text: "Tooltip Text",
```

Comment:
`Tooltip Text` might not belong here. I think it's only needed for `TooltipIcon`